modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,292 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+.The arrow shows the handle to use to reorder a mapping rule.
+image::cloudhub-dlb-mapping-rules-reorder.png[Handle to use to reorder a mapping rule]
+. Grab the handle to move the rule up or down in the list.
+. Click *Done Editing*.
+
+=== Change the Priority of Mapping Rules Using the CLI 
+
+To specify the mapping rule priority when creating a mapping rule with the CLI, use the `index` value.
+The rule at index `0` has the highest priority. The higher the index assigned, the lower the priority.
 
-You can define both input and output URLs using patterns or literal strings.
+See xref:anypoint-platform-cli-commands.adoc#cloudhub-load-balancer-mappings-add[cloudhub load-balancer mappings add].
 
-Mapping rules are attributes of the load balancer's SSL endpoint, which is identified by the certificate name.
-When you create a mapping rule, you must specify a certificate CN. Omitting the `[certificateName]` parameter adds the mappings to the default endpoint.
+=== Change the Priority of Mapping Rules Using the CloudHub API 
 
-If your SSL endpoint sets a https://en.wikipedia.org/wiki/Wildcard_certificate[wildcard certificate], and you want to use the subdomain portion in a mapping rule, you can use the pre-defined `{subdomain}` variable.
+When using the API to create a mapping rule, you cannot specify a priority order. 
+However, you can update the order of the rules later by sending a `PATCH` request to the load balancer endpoint:
 
-The rule which is defined first has high priority against other ones defined after it. This means that the first matched rule will be applied.
-You can create, view and delete existing rules using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-add[`mappings add`], xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-describe[`mappings describe`] and xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-remove[`mappings remove`] commands respectively.
+[source,text,linenums]
+----
+anypoint.mulesoft.com/cloudhub/api/organizations/{orgid}/loadbalancers/{loadbalancerId}
+----
 
+To determine the load balancer ID, perform a `GET` request to your endpoint:
 
-== Mapping Rule Examples
+[source,text,linenums]
+----
+/organizations/{orgid}/loadbalancers
+----
 
-The external load balancer domain name depends on the unique name you assign to it. In the following mapping rule examples, the load balancer is `lb-demo`.
 
-By default, your load balancer listens to external requests on HTTPS and communicates internally with your worker over HTTP. If you configured your Mule application within the VPC to listen on HTTPS, make sure you set `upstreamProtocol` to `HTTPS` when creating the mapping list using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-add[`load-balancer mappings add`] command.
+== Using a Literal 1:1 Mapping
 
-=== URL Mapping
+Instead of using a pattern, you can map literal app names.
+In this case, the rules must match the applications exactly.
```

Comment:
Is there more to say about this, such as relative pros and cons to the pattern vs. literal choice?
I guess this comes under the category of my need to understand the mapping methods earlier.

Reply:
The problem is...no one understands mapping rules...